app/services/form1010cg/service.rb

```diff
@@ -63,6 +63,9 @@ def process_claim_v2!
       [claim_pdf_path, poa_attachment_path].each { |p| File.delete(p) if p.present? }
 
       CARMA::Client::MuleSoftClient.new.create_submission_v2(payload)
+    rescue
+      Rails.logger.info "[Form 10-10CG] Submission failed for claim_guid: #{claim.guid}"
+      raise
```

Comment:
🤔 would it be useful to log the exception message together with the claim guid? might make it easier to debug if they're in the same place.
```suggestion
    rescue => e
      Rails.logger.info "[Form 10-10CG] Submission failed for claim_guid: #{claim.guid} - #{e.message}"
      raise e
```

Reply:
yeah, that seems like it could be valuable. I went ahead and added it in.